ide/coqide/coqOps.ml

```diff
@@ -333,9 +333,7 @@ object(self)
             try Doc.find document mem_marks
             with Not_found -> aux iter#backward_char in
         aux iter in
-      let ss =
-        find_all_tooltips s
-          (iter#offset - (buffer#get_iter_at_mark s.start)#offset) in
+      let ss = find_all_tooltips s iter#offset in
```

Comment:
While this is fixing the bug, this is also acknowledging that we're now using absolute offsets.

Reply:
I thought we wanted to fix this issue urgently.  This fix doesn't preclude modifying the design of some parts in the near future, but that will take considerably more time to analyze, discuss, code, test and review.  And, importantly, not introduce new bugs.  I will do some analysis on that fairly soon after addressing the urgent parts.